maint/local_python/binding_f08.py

```diff
@@ -507,15 +539,6 @@ def process_procedure(p):
         return (arg, arg)
 
     # ----
-    func_name = get_function_name(func)
-    if (need_cdesc(func)):
-        f08ts_name = func_name + "_f08ts"
-        c_func_name = re.sub(r'MPIX?_', r'MPIR_', func['name'] + '_cdesc')
-    else:
-        f08ts_name = func_name + "_f08"
-        c_func_name = re.sub(r'MPIX?_', r'MPIR_', func['name'] + '_c')
-    uses[c_func_name] = 1
-
```

Comment:
Note: this is moved up so the internal functions can directly use `func_name` and `c_func_name`. 

Background: `asan` and `ubsan` tests uses `gcc-9` which will enable `use_mpi_f08`. The `f08` tests were not active before this PR because they were disabled with previous configure and our Jenkins only tests `f08` with intel compilers -- thus not with `asan`, which catches this issue.